test/e2e/test-client.go

```diff
@@ -151,6 +152,17 @@ func CreateStudy(c api.ManagerClient) string {
 	return studyID
 }
 
+func DeleteStudy(c api.ManagerClient, studyID string) {
+	ctx := context.Background()
+	deleteStudyreq := &api.DeleteStudyRequest{
+		StudyId: studyID,
+	}
+	if _, err := c.DeleteStudy(ctx, deleteStudyreq); err != nil {
+		log.Fatalf("DeleteStudy error %v", err)
+	}
+	log.Printf("Study %s is deleted", studyID)
```

Comment:
You should check resources (study, trial, worker) are deleted by GetStudy or GetWorkers.

Reply:
Thanks! Done